packages/teams-js/test/public/sharing.spec.ts

```diff
@@ -23,8 +25,33 @@ describe('sharing_v1', () => {
     }
   });
 
+  Object.keys(FrameContexts)
+    .map(key => FrameContexts[key])
+    .forEach(frameContext => {
+      if (!allowedContext.includes(frameContext)) {
+        it(`should not allow calls from ${frameContext} context`, async () => {
+          await utils.initializeWithContext(frameContext);
+          const shareRequest: sharing.IShareRequest<sharing.IURLContent> = {
+            content: [
+              {
+                type: 'URL',
+                url: 'https://www.microsoft.com',
+                preview: true,
+                message: 'Test',
+              },
+            ],
+          };
+          expect(() => sharing.shareWebContent(shareRequest)).toThrowError(
+            `This call is only allowed in following contexts: ${JSON.stringify(
```

Comment:
Do we have tests that validate that the `sharing.shareWebContent` function cannot be called from disallowed contexts? (The reverse of these ones)

Reply:
Yes, we do already have them in place.